packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -165,7 +175,22 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
           }
         })
 
-        return connectionFromTasks(mappedIssues, first)
+        const nodes = mappedIssues.slice(0, first)
+        const edges = mappedIssues.map((node, index) => ({
+          cursor: `${index + afterInt}`,
```

Comment:
-2 this looks like an offset pagination rather than a cursor pagination. Offset paginations don't scale as well and can result in data being skipped or duplicated: https://stackoverflow.com/questions/18314687/how-to-implement-cursors-for-pagination-in-an-api

Instead, is there a unique field associated with each edge that we could use as a cursor?

Reply:
@nickoferrall yeah, but looks like Jira Server API supports only this kind of pagination: https://docs.atlassian.com/software/jira/docs/api/REST/7.6.1/ (see Pagination section)

Reply:
Ah how annoying. As we're going with an offset pagination, I'm curious why the cursor needs to be a string rather than a number?

Reply:
@nickoferrall relay requires this to be a string https://relay.dev/graphql/connections.htm#sec-Cursor. It throws an error  otherwise